mdn/src/main/java/org/apache/james/mdn/MDN.java

```diff
@@ -52,6 +60,7 @@
     public static class Builder {
         private String humanReadableText;
         private MDNReport report;
+        private boolean includeOriginalMessage;
```

Comment:
There is no such mention of a RFC. If anything we should have an `Optional<Message>` instead.

I will see with you tomorrow why you do need this, and help you get rid of it.